pkg/api/v20170701/types.go

```diff
@@ -98,19 +101,19 @@ const (
 
 // OrchestratorProfile contains Orchestrator properties
 type OrchestratorProfile struct {
-	OrchestratorType    OrchestratorType    `json:"orchestratorType"`
+	OrchestratorType    OrchestratorType    `json:"orchestratorType" validate:"required"`
 	OrchestratorVersion OrchestratorVersion `json:"orchestratorVersion"`
 }
 
 // MasterProfile represents the definition of master cluster
 type MasterProfile struct {
-	Count                    int    `json:"count"`
-	DNSPrefix                string `json:"dnsPrefix"`
-	VMSize                   string `json:"vmSize"`
-	OSDiskSizeGB             int    `json:"osDiskSizeGB,omitempty"`
+	Count                    int    `json:"count" validate:"required,eq=1|eq=3|eq=5"`
```

Comment:
It is from the cmd below. The package itself doesn't panic or fatal.

```
./cmd/deploy.go:                log.Fatalf("error parsing the api model: %s", err.Error())
./cmd/generate.go:              log.Fatalf("error parsing the api model: %s", err.Error())
./cmd/upgrade.go:               log.Fatalf("error parsing the api model: %s", err.Error())
```


Is something from our engine.go.